script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -227,22 +260,23 @@ Were it but the empty peace brought forth from a sense of loss.
 [sha:151ada9da39419f3a2fac9a3b96a8a4e7066f8d4]{
 -- Page 604, Offset 28758.
 -- 傷や思い出とかは残ったままだけど、とにかく、これで終わったんだ。明日からは普通の学生として、今まで通り平穏に生きていける。
-Injuries and memories will remain, but this is now over in any case. From tomorrow onward, I'll be able to lead a peaceful life as a normal student, just as I've done until now.
+Scars and memories will remain, but with this, it's all over.
+ From tomorrow on I'll be back to living life as an ordinary student, just as I did before.
 }
 [sha:dce77b6977c756f8501865927d2ff5b580063b6d]{
 -- Page 605, Offset 28759.
 -- 殺したり殺されたりするのはいい加減こりごりだ。
-I'm sick and tired of killing and being killed.
+I've had enough of people killing and being killed.
```

Comment:
Think it could lend itself to just be like this
>I've had enough of all this killing.

Sounds a bit stupid otherwise.